conf/documentation.conf

```diff
@@ -1875,3 +1875,25 @@ options=enabled|disabled
 description=<<EOT
 Show the parking portal to the device instead of the usual portal
 EOT
+
+[radius_authentication_methods.eap_authentication_types]
+type=multi
+options=GTC|MD5|MSCHAPv2|PEAP|FAST|LEAP
```

Comment:
I think we should eventually. IIRC there are stuff to configure for doing EAP-SIM that are outside of the scope of PacketFence. Let's add EAP-SIM in there when we actually handle this.